engine/installation/linux/oracle.md

```diff
@@ -14,9 +14,13 @@ To get started with Docker on Oracle Linux, make sure you
 
 ### OS requirements
 
-To install Docker, you need the 64-bit version of Oracle Linux 6 or 7, running
-the Unbreakable Enterprise Kernel Release 4 (4.1.12) or higher. This kernel
-supports the Docker `btrfs` storage engine.
+To install Docker, you need the 64-bit version of Oracle Linux 6 or 7.
```

Comment:
It would still be great to have a link to the official Oracle Linux Docker User Guide to help guide users to the installation process for the fully supported Docker release from Oracle.